pageserver/src/http/routes.rs

```diff
@@ -395,6 +395,9 @@ async fn timeline_create_handler(
                     format!("{err:#}")
                 ))
             }
+            Err(e @ tenant::CreateTimelineError::AncestorNotActive) => {
```

Comment:
Should be documented in openapi spec.

Reply:
Made into 503 as it is retryable assuming we don't hang in such state. Not documenting it.